src/export/dashboard.ts

```diff
@@ -0,0 +1,58 @@
+// (C) 2019 GoodData Corporation
+import { DashboardExport } from "@gooddata/typings";
+import { ApiResponse, ApiResponseError, XhrModule } from "../xhr";
+import { IExportResponse } from "../interfaces";
+import { handleHeadPolling, IPollingOptions } from "../util";
+import { sanitizeDateFilters, isDataExported } from "./util";
+
+interface IExportDashboardPayload {
+    dashboardExport: {
+        dashboardUri: string;
+        filters?: DashboardExport.FilterContextItem[];
+    };
+}
+
+export class DashboardModule {
+    constructor(private xhr: XhrModule) {}
+
+    public exportDashboardToPdf(
+        projectId: string,
+        dashboardUri: string,
+        filters: DashboardExport.FilterContextItem[] = [],
+        pollingOptions: IPollingOptions = {},
+    ): Promise<IExportResponse> {
+        const convertedDateFilters = sanitizeDateFilters(filters);
```

Comment:
How about Attribute Filters?

Reply:
In `sanitize` function, we only need to convert `from|to` from `number` to `string` to match backend specification. So that we touch nothing to attribute filters.